app/components/environment/environment-service.es6

```diff
@@ -57,7 +57,7 @@ class Environment {
             SG: segmentEnvs,
             SG2: segmentEnvs,
             STAGE2: segmentEnvs,
-            PS: ['QA Call-Center', 'QA Admin', 'Stage Call-Center', 'Stage Admin'],
+            PS: ['Devco Salesforce', 'SA QA Call-Center', 'SF QA Call-Center', 'SA Stage Call-Center', 'SF Stage Call-Center', 'SA QA Admin', 'SA Stage Admin'],
```

Comment:
line length: do you mind pulling this out into its own variable and putting each item on its own line? 

similar to how hcaEnvs has its on array up above.